tests/operators/test_python.py

```diff
@@ -847,6 +847,16 @@ def default_kwargs(*, python_version=sys.version_info[0], **kwargs):
         kwargs["python_version"] = python_version
         return kwargs
 
+    @mock.patch("airflow.operators.python.importlib")
+    def test_virtuenv_not_installed(self, importlib):
+        importlib.util.find_spec.return_value = None
+        with pytest.raises(AirflowException):
```

Comment:
CAn you please add `match=` and add the message regexp?

Reply:
Done